template/docs/conf.py.jinja

```diff
@@ -59,8 +61,8 @@ autodoc_type_aliases = {
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
-    'numpy': ('https://numpy.org/doc/stable/', None),
-    'scipp': ('https://scipp.github.io/', None),
+    'numpy': ('https://numpy.org/doc/stable/', None),{% if orgname == 'scipp'  %}
+    'scipp': ('https://scipp.github.io/', None),{% endif %}
```

Comment:
How about this...?

```suggestion
    'numpy': ('https://numpy.org/doc/stable/', None),
    '{{orgname}}': ('https://{{orgname}}.github.io/', None),
```

Reply:
Don't see why? What are the odds that someone has org-wide github pages and wants to use this template?

Reply:
Then what's the reason of having this if condition here...?

Reply:
Because Scipp has it, and (almost) all other projects in the org use it?

Reply:
Apart from this, it looks good to me...!